src/components/OutgoingComponent.jsx

```diff
@@ -0,0 +1,46 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import { Table, TableBody, TableCell, TableHead, TableRow, Paper } from '@material-ui/core';
+
+class OutgoingComponent extends Component {
+
+    static propTypes = {
+        outgoings: PropTypes.array.isRequired,
+    };
+
+    componentDidMount() {
+    }
+
+    render() {
+        return (
+            <Paper className="container">
+                <br /><br /><br />
```

Comment:
Wie du bereits in der E-Mail geschrieben hast, müssen wir das Verhalten von Header und Inhalt anders lösen. Auch die Menüleiste muss anders positioniert werden.

Reply:
ich glaube das müssen wir in einer anderen Story umsetzen oder?

Reply:
#20